src/coreclr/tools/aot/ILCompiler.Diagnostics/ISymNGenWriter.cs

```diff
@@ -1,7 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-#pragma warning disable 436 // SuppressUnmanagedCodeSecurityAttribute defined in source and mscorlib
+#pragma warning disable 436 // SuppressUnmanagedCodeSecurityAttribute defined in source and SPCL
```

Comment:
Is this pragma disable still relevant? Can we just delete it?

Reply:
Ugh. Yes, this should just be removed.